framework/auth/views.py

```diff
@@ -230,9 +239,16 @@ def send_confirm_email(user, email):
     except NoResultsFound:
         merge_target = None
 
+    campaign = campaigns.campaign_for_user(user)
+    # Choose the appropriate email template to use
+    if merge_target:
+        mail_template = mails.CONFIRM_MERGE
+    else:
```

Comment:
`campaigns.email_template_for_campaign(campaign, default=mails.CONFIRM_EMAIL)` 
takes a default fallback template to handle falsy values of campaign.